style.css

```diff
@@ -93,3 +93,394 @@ h6 {
+  line-height: 2rem;
+  margin-left: -1rem;
+  padding: 0 0.7rem;
+}
+
+.grid-container2 {
+  background-image: url(/image/placeholder.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  grid-template-columns: repeat(auto-fit, minmax(100%, 1fr));
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  grid-gap: 18vh;
+  padding: 18vh 4vw;
+  margin-top: -11rem;
+}
+
+.third_h1 {
+  display: flex;
+  justify-content: center;
+  padding-top: 3rem;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 35px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.third_p {
+  display: flex;
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.middle-button2 {
+  display: flex;
+  justify-content: center;
+}
+
+.css-button2 {
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 3rem;
+  margin-left: -1rem;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  padding: 0 0.7rem;
+}
+
+.html-button2 {
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 3rem;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  margin-left: 0.5rem;
+  padding: 0 0.7rem;
+}
+
+.bootstrap-button2 {
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 3rem;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  margin-left: 0.5rem;
+  padding: 0 0.7rem;
+}
+
+.project-button2 {
+  cursor: pointer;
+  width: 327px;
+  height: 48px;
+  background-color: #ff6b00;
+  color: #fff;
+  border: none;
+  margin-left: 0.19rem;
+  margin-top: 1.5rem;
+  font-family: 'Inter', sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  font-style: normal;
+  line-height: 24px;
+}
+
+.about-me {
+  background-color: #1c1a19;
+  width: 375px;
+  height: 600px;
+  background-image: url(/image/Group2.png);
+  background-repeat: no-repeat;
+  background-position: 100% 100%;
+}
+
+.line-1 {
+  height: 1px;
+  width: 300px;
+  background-color: #fff;
+  margin-left: 2rem;
+  margin-top: -5rem;
+}
+
+.ignore {
+  color: #1c1a19;
+}
+
+.about-section-h1 {
+  display: flex;
+  justify-content: center;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 70px;
+  line-height: 88px;
+  padding-top: 4rem;
+  padding-right: 1.6rem;
+  padding-left: 1.6rem;
+}
+
+.about-section-paragraph {
+  display: flex;
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
```

Comment:
- [ ] This is okay that you have aligned the content with flexbox for the `About me` section, But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times

Reply:
I have fixed this. Thank you